Pincode implementation

rip32700 opened the pull request.

rip32700 commented:
Fixes #12

maratsubkhankulov commented:
Looking good, but lets not ask user to input pin on disable.

michaelScoff commented:
@maratsubkhankulov @magicaltramp why? You need to enter passcode to disable it

rip32700 commented:
@michaelScoff Marat thought it would be better without asking passcode to disable it since it is done like this in the iOS version and thus for consistency. I personally think it is a good idea to ask for it when disabling it, what do you think?

michaelScoff commented on app/src/main/java/com/wallet/crypto/trustapp/views/CustomPinActivity.java:
```diff
@@ -0,0 +1,41 @@
+        } else {
+            builder = new AlertDialog.Builder(this);
+        }
+        builder.setTitle("Warning")
```
Why do we need to contact devs? I don’t think we can help here 

Also strings should be added always to the special file, like android devs do

michaelScoff commented:
Actually you are right, let’s keep it simple for now

rip32700 commented:
@maratsubkhankulov I changed to not ask for pincode when disabling
@michaelScoff I removed forgot dialog

@maratsubkhankulov @madcake It should work now with the latest master. Please review and test.
@madcake can you test fingerprint on a real device?

maratsubkhankulov commented:
@rip32700 thanks Phil.
